tests/Unit/LocalFileVolumeReadOnlyTest.php

```
@@ -0,0 +1,219 @@
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Helper function to parse volumes and detect read-only status.
+ * This mirrors the logic in LocalFileVolume::isReadOnlyVolume()
+ *
+ * Note: We match on mount_path (container path) only, since fs_path gets transformed
+ * from relative (./file) to absolute (/data/coolify/services/uuid/file) during parsing
+ */
+function isVolumeReadOnly(string $dockerComposeRaw, string $serviceName, string $mountPath): bool
+{
+    $compose = Yaml::parse($dockerComposeRaw);
```

PHPMD (2.15.0)

Avoid using static access to class '\Symfony\Component\Yaml\Yaml' in method 'isVolumeReadOnly'. (undefined)

(StaticAccess)